vignettes/Overview.Rmd

```diff
@@ -0,0 +1,398 @@
+read in to memory in their entirety using `readFastq`; we do
+this for our sample data
+```{r readFastq}
+fq <- readFastq(fl)
+```
+The result of data input is an instance of class `ShortReadQ`
+
+|                    |                                                   |
+| ------------------ | ------------------------------------------------- |
+| DNAStringSet       | (`r Biocpkg("Biostrings")`) Short read sequences |
+| FastqQuality, etc. | Quality encodings                                 |
+| ShortReadQ         | Reads, quality scores, and ids                    |
+
+**Table 2:** Primary data types in the `r Biocpkg("ShortRead")` package.
+
+This class contains reads, their quality scores, and optionally the id
+of the read.
+```{r ShortReadQ}
+fq
+fq[1:5]
+head(sread(fq), 3)
+head(quality(fq), 3)
+```
+The reads are represented as _DNAStringSet_
+instances, and can be manipulated with the rich tools defined in the
+`r Biocpkg("Biostrings")` package. The quality scores are represented by a
+class that represents the quality encoding inferred from the file; the
+encoding in use can be discovered with
+```{r encoding}
+encoding(quality(fq))
+```
+The primary source of documentation for these classes is
+`?ShortReadQ` and `?QualityScore`.
+
+# Common work flows
+
+
+## Quality assessment
+
+FASTQ files are often used for basic quality assessment, often to
+augment the purely technical QA that might be provided by the
+sequencing center with QA relevant to overall experimental design. A QA report is generated by creating a vector of paths to FASTQ files
+```{r qa-files1, eval=FALSE}
+fls <- dir("/path/to", "*fastq$", full=TRUE)
+```
+collecting statistics over the files
+```{r qa-qa, eval=FALSE}
+qaSummary <- qa(fls, type="fastq")
+```
+and creating and viewing a report
+```{r qa-view, eval=FALSE}
+browseURL(report(qaSummary))
+```
+By default, the report is based on a sample of 1M
+reads.
+These QA facilities are easily augmented by writing custom functions
+for reads sampled from files, or by exploiting the elements of the
+object returned from `qa()`, e.g., for an analysis of
+ArrayExpress experiment E-MTAB-1147:
+```{r qa-files2, echo=FALSE}
+load("qa_E-MTAB-1147.Rda")
+```
+
+```{r qa-elements}
+qaSummary
+```
+For instance, the count of reads in each lane is summarized in the
+`readCounts` element, and can be displayed as
+```{r qa-readCounts}
+head(qaSummary[["readCounts"]])
+head(qaSummary[["baseCalls"]])
+```
+The `readCounts` element contains a data frame with 1 row and
+3 columns (these dimensions are indicated in the parenthetical
+annotation of `readCounts` in the output of
+`qaSummary`). The rows represent different lanes. The columns
+indicated the number of reads, the number of reads surviving the
+Solexa filtering criteria, and the number of reads aligned to the
+reference genome for the lane. The `baseCalls` element
+summarizes base calls in the unfiltered reads.
+The functions that produce the report tables and graphics are 

```

Comment:
Insert a blank line here so that the HTML contains a new paragraph, like the Rnw.

There are other places in the Rmd that need to be modified.

Reply:
Done

Reply:
I think we've caught the other locations.